### PR TITLE
Fix fetch data on cache miss in Repository.java

### PR DESCRIPTION
**Pull request by alvarez-fabian**
# Summary
Fix fetch data on cache miss in Repository.java

### CODE REVIEW

File: networking/src/main/java/ar/com/wolox/wolmo/networking/offline/Repository.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public Repository(@NonNull C cache) {
      * @param queryStrategy that determines how to react to local/network actions
      */
     public Query<T> query(@AccessPolicy final int policy, @NonNull final Call<T> call,
-                          @NonNull final QueryStrategy<T, C> queryStrategy) {
+            @NonNull final QueryStrategy<T, C> queryStrategy) {
```
Comment (emalamela) [inline networking/src/main/java/ar/com/wolox/wolmo/networking/offline/Repository.java]: Why did you remove this spacing? It existed to align the parameters.

**Review by SpicyCactuar — CHANGES_REQUESTED** [review]
Just 1 change, nice bug-catcher @alvarez-fabian !

File: networking/src/main/java/ar/com/wolox/wolmo/networking/offline/Repository.java (hunk 1 above)
Comment (alvarez-fabian) [reply networking/src/main/java/ar/com/wolox/wolmo/networking/offline/Repository.java]: Lol, auto-formatting

**Review by SpicyCactuar — APPROVED** [review]
No GIF, Mobile OP.

Great finding!
